app/models/ext_management_system.rb

```diff
@@ -542,6 +544,14 @@ def total_vms_never;     vm_count_by_state("never");     end
 
   def total_vms_suspended; vm_count_by_state("suspended"); end
 
+  def supports_block_storage
```

Comment:
maybe because you've added the alias *before* the block storage mixing got included?
the supports_feature_mixin defaults all features to unsupported. once it's supported it defines another method.

I tried to remove this in https://github.com/ManageIQ/manageiq/pull/11322 - but unfortunately that PR got stale